run_dir/static/js/index.js

```diff
@@ -0,0 +1,137 @@
+fill_quotas_table = function() {
+
+   var uppmax_projects_string = $('#indexpage-js-import').data('uppmax_projects');
+    // make a normal array from string
+    // but there must be a better way
+    var uppmax_projects = uppmax_projects_string.split(',');
+    $.each(uppmax_projects, function(i, project_id){
+        $('#project_quotas tbody').append('<tr><td>' + project_id + '</td><td id="sparkline_quota_' + project_id + '" class="plot-column"><div></div></td></tr>');
+         // Load the data
+        $.getJSON("/api/v1/quotas/" + project_id, function(api_data) {
+            // Massage the data
+            var raw_data = api_data[0]["data"];
+            var plot_data = [];
+            var max_value = 0;
+            var quota_percent;
+            var current_quota;
+            $.each(raw_data, function(i, point){
+                point.y /= 1000000000000;
+                point.limit /= 1000000000000;
+                current_quota = point.limit
+                quota_percent = (100 * point.y / point.limit).toFixed(1); 
+                plot_data.push([point.x * 1000, point.y, point.percent]);
+                if(max_value < point.y) { max_value = point.y; }
+            });
+            console.log(plot_data)
+            var background = '';
```

Comment:
I know that this is a leftover from the previous structure of the code, but this would probably be clearer:

```javascript
if (quota_percent > 90) {
  $('#sparkline_quota_'+project_id).parent().addClass('danger');
} else if (quota_percent > 80) {
  $('#sparkline_quota_'+project_id).parent().addClass('warning');
}
```